### DISCUSSION TITLE
Adding other (than Postgis) extensions to the image?

### DISCUSSION THREAD

**Discussion by Brent-Wood** [postgis/docker-postgis#263] [Ideas]
Has there been any discussion regarding adding other (than Postgis) Postgres extensions to the Docker image?
I'm particularly interested in postgres_fdw and ogr_fdw (but have others in mind as well).

The first is usually provided with core postgres, the second is a very useful extension to use with Postgis.

Thanks

Brent Wood

**Comment by submarcos** [comment]
It would be great. For me I cant use anymore this image because 3+ has not raster activated. This is a breaking changes from 2.5+ https://github.com/postgis/docker-postgis/issues/265

**Comment by phillipross** [comment]
At some point I imagine there could be the notion of docker images in this repo that focus more on providing out of the box features at the cost of image size.  Currently the images are based on the official postgres docker images, and would or would not include the same non-postgis extensions.  These images are also based on debian which stopped including raster by default.  Rather than diverging from upstream patterns, it will be more maintainable to continue following upstream patterns and provide documentation on ways users can extend the images.
